test/e2e/apimachinery/flowcontrol.go

```diff
@@ -74,27 +74,31 @@ var _ = SIGDescribe("API priority and fairness", func() {
 	// higher QPS client cannot drown out the other one despite having higher
 	// priority.
 	ginkgo.It("should ensure that requests can't be drowned out (priority)", func() {
-		flowSchemaNamePrefix := "e2e-testing-flowschema"
-		priorityLevelNamePrefix := "e2e-testing-prioritylevel"
+		flowSchemaNamePrefix := "e2e-testing-flowschema-" + f.UniqueName
+		priorityLevelNamePrefix := "e2e-testing-prioritylevel-" + f.UniqueName
 		loadDuration := 10 * time.Second
+		highQPSClientName := "highqps-" + f.UniqueName
+		lowQPSClientName := "lowqps-" + f.UniqueName
+
 		type client struct {
-			username              string
-			qps                   float64
-			priorityLevelName     string
-			concurrencyMultiplier float64
-			concurrency           int32
-			flowSchemaName        string
-			matchingPrecedence    int32
-			completedRequests     int32
+			username                    string
+			qps                         float64
+			priorityLevelName           string
+			concurrencyMultiplier       float64
+			concurrency                 int32
+			flowSchemaName              string
+			matchingPrecedence          int32
+			completedRequests           int32
+			expectedCompletedPercentage float64
 		}
 		clients := []client{
 			// "highqps" refers to a client that creates requests at a much higher
 			// QPS than its counter-part and well above its concurrency share limit.
 			// In contrast, "lowqps" stays under its concurrency shares.
 			// Additionally, the "highqps" client also has a higher matching
 			// precedence for its flow schema.
-			{username: "highqps", qps: 100.0, concurrencyMultiplier: 2.0, matchingPrecedence: 999},
-			{username: "lowqps", qps: 5.0, concurrencyMultiplier: 0.5, matchingPrecedence: 1000},
+			{username: highQPSClientName, qps: 100.0, concurrencyMultiplier: 2.0, matchingPrecedence: 999, expectedCompletedPercentage: 0.75},
```

Comment:
Was there any quantitative reasoning for picking 100 here?  If not and we just want this test to stop flaking, why not try a lower number?

Reply:
mmm i guess we chose 100 for the highqps client as a random number? @adtac

Reply:
there isn't much reasoning behind 100 other than that it's a nice round number that's large enough -- I don't have any objections to reducing the QPS a bit (as long as the ratio between the two clients stays large enough)

Reply:
So #96874 is mainly about reducing the expected as well as attempted throughput.